src/shared/locales/en/translation.json

```diff
@@ -63,6 +63,8 @@
         "password": "Password",
         "unrecognisedPassword": "Unrecognised password",
         "unrecognisedPasswordExplanation": "The password was not recognised. Please try again.",
+        "unrecognizedAccount": "Unrecognized account",
+        "unrecognizedAccountExplanation": "Account named {{accountName}} was not recognized. Consider reset your wallet.",
```

Comment:
IOTA aims to use UK English as much as possible, so `unrecognized` should be `unrecognised` and `recognized` should be `recognised`.